scripts/commands.py

```diff
@@ -298,4 +298,4 @@ def delete_agent(key):
     result = agents.delete_agent(key)
     if not result:
         return f"Agent {key} does not exist."
-    return f"Agent {key} deleted."
+    return f"Agent {key} deleted."
```

Comment:
This is optional, but add an extra line feed at the end of file to remove this (-) sign from the diff. I will still approve if you won't